src/ctapipe/tools/make_irf.py

```diff
@@ -0,0 +1,615 @@
+
+    proton_target_spectrum = traits.UseEnum(
+        Spectra,
+        default_value=Spectra.IRFDOC_PROTON_SPECTRUM,
+        help="Name of the spectra used for the simulated proton spectrum",
+    ).tag(config=True)
+
+    electron_file = traits.Path(
+        default_value=None,
+        allow_none=True,
+        directory_ok=False,
+        help="Electron input filename and path",
+    ).tag(config=True)
+
+    electron_target_spectrum = traits.UseEnum(
+        Spectra,
+        default_value=Spectra.IRFDOC_ELECTRON_SPECTRUM,
+        help="Name of the spectra used for the simulated electron spectrum",
+    ).tag(config=True)
+
+    chunk_size = Integer(
+        default_value=100000,
+        allow_none=True,
+        help="How many subarray events to load at once while selecting.",
+    ).tag(config=True)
+
+    output_path = traits.Path(
+        default_value="./IRF.fits.gz",
+        allow_none=False,
+        directory_ok=False,
+        help="Output file",
+    ).tag(config=True)
+
+    obs_time = AstroQuantity(
+        default_value=u.Quantity(50, u.hour),
+        physical_type=u.physical.time,
+        help="Observation time in the form ``<value> <unit>``",
+    ).tag(config=True)
+
+    edisp_parameterization = traits.ComponentName(
+        EnergyMigrationMakerBase,
+        default_value="EnergyMigration2dMaker",
+        help="The parameterization of the energy migration to be used.",
+    ).tag(config=True)
+
+    aeff_parameterization = traits.ComponentName(
+        EffectiveAreaMakerBase,
+        default_value="EffectiveArea2dMaker",
+        help="The parameterization of the effective area to be used.",
+    ).tag(config=True)
+
+    psf_parameterization = traits.ComponentName(
+        PsfMakerBase,
+        default_value="Psf3dMaker",
+        help="The parameterization of the point spread function to be used.",
+    ).tag(config=True)
+
+    bkg_parameterization = traits.ComponentName(
+        BackgroundRateMakerBase,
+        default_value="BackgroundRate2dMaker",
+        help="The parameterization of the background rate to be used.",
+    ).tag(config=True)
+
+    energy_bias_res_parameterization = traits.ComponentName(
+        EnergyBiasResolutionMakerBase,
+        default_value="EnergyBiasResolution2dMaker",
+        help=(
+            "The parameterization of the bias and resolution benchmark "
+            "for the energy prediction."
+        ),
+    ).tag(config=True)
+
+    ang_res_parameterization = traits.ComponentName(
+        AngularResolutionMakerBase,
+        default_value="AngularResolution2dMaker",
+        help="The parameterization of the angular resolution benchmark.",
+    ).tag(config=True)
+
+    sens_parameterization = traits.ComponentName(
+        SensitivityMakerBase,
+        default_value="Sensitivity2dMaker",
+        help="The parameterization of the point source sensitivity benchmark.",
+    ).tag(config=True)
+
+    full_enclosure = Bool(
+        False,
```

Comment:
I think the option should be `point_like` with default `False` so that the default are non-point-like / full-enclosure IRFs.

Reply:
I think I changed that already twice in the past. I would agree that doing it this way would make the most sense, if only to be consistent with pyirf. @Tobychev Are you fine with changing it like this?